docs/dev-guide/installingNative.md

```diff
@@ -39,14 +39,6 @@ Most of these prerequisites can be installed using the package manager of your O
     > sudo service postgresql start
     > ```
 
-    Python 3.10 is not available in the default Ubuntu repositories -- you can install it through the [deadsnakes PPA](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa):
-
-    ```sh
-    sudo add-apt-repository ppa:deadsnakes/ppa
-    sudo apt update
-    sudo apt install python3.10 python3.10-dev
-    ```
-
```

Comment:
I believe by removing this you also remove the prerequisites to install uv, namely pip.

Reply:
Ah -- uv can be installed without pip. I'd rather recommend their standalone installer.

Reply:
Done in [ca75626](https://github.com/PrairieLearn/PrairieLearn/pull/13628/commits/ca75626757549968a3bf3abb0d0d93be7ae7a4eb).